Marsiple/Common/Network/PostsFetcher.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  PostsFetcher.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 17/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+struct PostsFetcher {
+    func get(succes: @escaping ([Post]) -> Void, faliure: @escaping (String) -> Void) {
```

Comment:
Quite a few typos in this method definition, please be more attentitive. 

`succes` -> `success` | `faliure` -> `failure`